src/rtos/rtos-chibios.ts

```diff
@@ -1,11 +1,13 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+import { breadcrumbTemplate } from '@microsoft/fast-foundation';
+import { buffer } from 'stream/consumers';
```

Comment:
`buffer ` and `breadcrumbTemplate` do not seem to be used or do i overse something?

Reply:
removed